packages/uploadthing/src/internal/incompat-node-guard.ts

```diff
@@ -23,7 +23,7 @@ export function incompatibleNodeGuard() {
   if (!major || !minor) return;
 
   // Require ^18.13.0
-  if (!(major >= 18 && minor >= 13)) {
+  if (!((major === 18 && minor >= 13) || major >= 19)) {
```

Comment:
It's just my personal preference, but wouldn't be more readable if we remove negation and invert the condition? Like

```suggestion
// In case node >= 18.13 we return and allow further execution
if ((major === 18 && minor >= 13) || major >= 19) return;

console.error(/* error goes here */)
process.exit(1)
```

Reply:
good idea!